app/components/optimized-power-select/options/component.js

```diff
@@ -0,0 +1,111 @@
+import { action } from "@ember/object";
+import Component from "@glimmer/component";
+
+const isTouchDevice = !!window && "ontouchstart" in window;
```

Comment:
This seems to be copied form [the original eps options file](https://github.com/cibernox/ember-power-select/blob/64e91f272f2df87fe1cd5472abf596abfaa42ab1/addon/components/power-select/options.ts#L13)